app/embeds/index.js

```diff
@@ -57,6 +58,13 @@ export default [
     component: Airtable,
     matcher: matcher(Airtable),
   },
+  {
+    title: "ClickUp",
+    keywords: "project",
+    icon: () => <Img src="/images/clickup.png" />,
```

Comment:
![image](https://user-images.githubusercontent.com/380914/90590067-6c80a100-e194-11ea-8348-4580fb8a8e78.png)

Can we get a version of the logo on a transparent background